packages/esbuild/src/generators/init/init.ts

```diff
@@ -5,14 +5,15 @@ import {
   Tree,
 } from '@nrwl/devkit';
 import { Schema } from './schema';
-import { esbuildVersion } from '@nrwl/js/src/utils/versions';
+import { esbuildVersion, npmdtsVersion } from '@nrwl/js/src/utils/versions';
 
 export async function esbuildInitGenerator(tree: Tree, schema: Schema) {
   const task = addDependenciesToPackageJson(
     tree,
     {},
     {
       esbuild: esbuildVersion,
+      'npm-dts': npmdtsVersion,
```

Comment:
Lets not install this by default. The error logging that I mentioned above should tell folks to install it if needed.